ProPublicaCongressAPI/InternalModels/SpecificBill.cs

```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
         [JsonProperty("sponsor")]
         public string SponsorMemberName { get; set; }
 
+        [JsonProperty("sponsor_id")]
+        public string SponsorId { get; set; }
+
         [JsonProperty("sponsor_uri")]
         public string SponsorMemberDetailUrl { get; set; }
 
+        [JsonProperty("sponsor_party")]
+        public string SponsorParty { get; set; }
+
+        [JsonProperty("sponsor_state")]
+        public string SponsorState { get; set; }
+
         [JsonProperty("gpo_pdf_uri")]
         public string BillDocumentPdfUrl { get; set; }
 
+        [JsonProperty("congressdotgov_url")]
+        public string CongressDotGovUrl { get; set; }
+
+        [JsonProperty("govtrack_url")]
+        public string GovTrackUrl { get; set; }
+
         [JsonProperty("introduced_date")]
         public string DateIntroduced { get; set; }
 
+        [JsonProperty("active")]
+        public bool Active { get; set; }
+
+        [JsonProperty("house_passage")]
+        public string HousePassage { get; set; }
+
+        [JsonProperty("senate_passage")]
+        public string SenatePassage { get; set; }
+
+        [JsonProperty("enacted")]
+        public string Enacted { get; set; }
+
+        [JsonProperty("vetoed")]
+        public string Vetoed { get; set; }
+
         [JsonProperty("cosponsors")]
         public int CosponsorCount { get; set; }
 
+        [JsonProperty("withdrawn_cosponsors")]
+        public int WithdrawnCosponsorCount { get; set; }
+
         [JsonProperty("primary_subject")]
         public string PrimarySubject { get; set; }
 
         [JsonProperty("committees")]
         public string Committees { get; set; }
 
+        [JsonProperty("committee_codes")]
```

Comment:
I was going off of what I saw in fiddler:

![image](https://user-images.githubusercontent.com/438824/27565717-effa1ea6-5aad-11e7-8a36-876b7f0544bf.png)

Given they aren't documented, I'm happy to remove them.  Let me know what you'd like me to do there